website/content/upgrading/from_v1_to_v2.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Upgrade from v1 to v2
+---
+
+Dkron v2 brings lots of changes to the previous version. To successfully upgrade from v1 to v2 you need to take care of certain changes:
+
+## Migrating Jobs
+
+To migrate jobs from v1 to v2 they should be exported from the v1 cluster and imported into v2.
+
+A basic script to do that can be found [here](https://gist.github.com/pjz/94f4bd81a0897fd64db44593078e2156)
+
+You can take the chance to change your job tags as explained in the following section.
+
+## Changing tags and metadata
+
+Tags behaves different in dkron 2. There is several dkron tags that changes its name and becomes reserved, the reserved tags are:
```

Comment:
Tags behave different in dkron 2. There are several dkron tags that changed names and have become reserved. The reserved tags are: